consensus/src/round_manager_test.rs

```diff
@@ -1802,261 +1852,6 @@ fn block_retrieval_test() {
     });
 }
 
-#[test]
-pub fn forking_retrieval_test() {
```

Comment:
the test works when order vote is disabled?

Reply:
Yes. Earlier, we removed syncing forks entirely even when order votes are disabled. As we are no longer doing that, I added the test back.